repo.json

```diff
@@ -2041,6 +2041,11 @@
           "version": "0.1.0",
```

Comment:
Is it necessary to keep the version 0.1.0 there?

Reply:
Current commit points to the version before the dist directory was updated.
It is probably better that the version 0.1.0 refers the commit of f6dd083c0c6a241f6a6b82f4f01cbf702ddeaf72.

Reply:
You are right 0.1.0 is not really necessary since there are no breaking changes. 
Fixed with the PR, for which I am really grateful.